src/main/resources/application.yaml

```diff
@@ -209,19 +209,19 @@ uk:
             JOINT_APPLICANT1_APPLICANT1_CHANGES_MADE: 'f4971c37-45ca-470a-9ae4-d64cd96b73ff'
             JOINT_APPLICANT2_APPLICANT1_CHANGES_MADE: 'aa5ae136-7004-46f6-81ad-606242f4a501'
             JOINT_APPLICANT2_APPLICANT1_CHANGES_MADE_SOLICITOR: 'd050d274-c0d0-4ce3-bb31-4a9f3ad1b7f1'
-            JOINT_APPLICANT2_SOLICITOR_APPLICANT1_CHANGES_MADE: '6a6eb8ca-17e8-49a7-adf6-6f3cd4e1b546'
-            JOINT_APPLICATION_ACCEPTED: 'a311523f-be00-47de-ba8d-8afd1e3ccd38'
-            JOINT_APPLICATION_OVERDUE: '8cc94ef3-2507-41cd-8267-8c9857d41505'
+            JOINT_APPLICANT2_SOLICITOR_APPLICANT1_CHANGES_MADE: '84a20001-8149-499d-9558-b6884697e195'  #TODO: Update Welsh translation
+            JOINT_APPLICATION_ACCEPTED: 'c6f1a5b1-da8b-4da6-952d-a373aeaf26c6'
+            JOINT_APPLICATION_OVERDUE: '5b1b6d14-6a3c-4de8-a498-ea92187e7f59'
             JOINT_APPLICATION_SUBMITTED: '2b606152-85f4-43fc-9e2f-48edfd15f077'
-            JOINT_APPLICATION_APPROVED_APPLICANT1_REMINDER: '7d8db5c5-4b9c-4221-8fed-8c542d45a6af'
-            SOLE_APPLICANT_APPLICATION_ACCEPTED: '80ec944d-2791-42ec-9551-1f76da6f7eb4'
-            SOLE_APPLICANT_PARTNER_HAS_NOT_RESPONDED: '0c8de3fa-2294-41d0-8060-3de20bc23712'
-            SOLE_RESPONDENT_APPLICATION_ACCEPTED: 'ddb52f9e-64d4-43a9-817d-405db378d4c5'
+            JOINT_APPLICATION_APPROVED_APPLICANT1_REMINDER: '449e92ba-f68d-4a06-9796-50ca87fb4827'
+            SOLE_APPLICANT_APPLICATION_ACCEPTED: 'cc2386c3-688f-4bc0-b7cd-abe9cfbf53c1'
+            SOLE_APPLICANT_PARTNER_HAS_NOT_RESPONDED: '17da4bc0-66c5-4617-a8a9-069421e526ab'
+            SOLE_RESPONDENT_APPLICATION_ACCEPTED: '067e2e14-fb8b-4274-a5d4-1db91b73cab2'
             OVERSEAS_RESPONDENT_HAS_EMAIL_APPLICATION_ISSUED: '1162cc6f-3d04-4d1d-8240-4754c5242803'
```

Comment:
Do we need to update the Welsh for `OVERSEAS_RESPONDENT_HAS_EMAIL_APPLICATION_ISSUED` too?

Reply:
Thanks sorry i missed that one and have updated it now